Makefile

```diff
@@ -0,0 +1,23 @@
+###
+### Variables
+###
+ANSIBLE_VERSION=2.5
+
+
+###
+### Default
+###
+help:
+	@printf "%s\n\n" "Available commands"
+	@printf "%s\n"   "make test             Test the Ansible role"
+	@printf "%s\n"   "make lint             Lint source files"
+	@printf "%s\n"   "make help             Show help"
+
+test:
+	docker run --rm -it \
```

Comment:
given that this is an AWS role, we'll need to also expose the credentials (preferably as env vars).